### DISCUSSION TITLE
alist和qb是不是要在同一个机器上

### DISCUSSION THREAD

**Discussion by windyhun** [AlistGo/alist#5067] [Q&A] [answered]
我想问下qb服务器是不是需要和alist服务在同一个机器上，因为我路由器上面挂着alist（alist在docker里面并且映射了/opt/alist/data/temp/aria2
/opt/alist/data/temp/qbittorrent），
服务器上挂着qb，用离线下载就会上传失败，显示failed to open file /opt/alist/data/temp/qbittorrent/3b058078-b251-4755-b48c-213848ade1bb/xxx.zip: open /opt/alist/data/temp/qbittorrent/3b058078-b251-4755-b48c-213848ade1bb/xxx.zip: no such file or directory

在qb服务器里面可以看到有正常的下载，qb服务器里面的/opt/alist/data/temp/qbittorrent/里面也有文件

**Comment by windyhun** [comment]
有人吗

**Comment by xhofe** [comment, marked as answer]
是
